exercises/grains/canonical-data.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "comments": [
     "The final tests of square test error conditions",
-    "The expection for these tests is -1, indicating an error",
+    "The exception for these tests is -1, indicating an error",
```

Comment:
I think it is expectation rather than exception

Reply:
Possible :smile: